lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
See, that's what I thought as well. But for whatever reason, the default Promises don't seem to handle errors thrown within the Promise chain the same way.  I had originally tried doing the same thing, without bringing Bluebird in, but the errors were just swallowed